### DISCUSSION TITLE
Missing documentation about usage outside Laravel in Google Play Billing.

### DISCUSSION THREAD

**Discussion by ShreeSoftech-Rupesh** [imdhemy/laravel-in-app-purchases#387] [General]
I attempted to install your package in my CodeIgniter project, but I encountered an issue: the setup instructions for Google Play Billing are missing.

**Comment by mattli001** [comment]
Recently, I have been using packages to integrate Google Play Console's IAP. Here are two detailed configuration documents for reference:

1. [Google Play Service Account Key](https://documentation.qonversion.io/docs/service-account-key-android#3-link-google-cloud-project) (last updated 2024-05)
2. [Android In-App Products Setup](https://documentation.qonversion.io/docs/android-in-app-products) (last updated 2024-01)

Especially when encountering error
```json
{
 "error": {
  "code": 401,
  "message": "The current user has insufficient permissions to perform the requested o (truncated...)"
}
```
it was by referring to the two documents that the problem was solved.
